src/queries.ts

```diff
@@ -102,6 +104,7 @@ function isMultiParams(object: any): object is MultiQueryParams {
 export function canonicalEvents(params: DesktopQueryParams | AndroidQueryParams): string {
   // Needs escaping for regex patterns like '\w' to work (JSON.stringify adds extra unecessary escaping)
   const categories_str = JSON.stringify(params.categories).replace(/\\\\/g, '\\');
+  const neverTreatAsAfkFilter_str = isDesktopParams(params)? JSON.stringify(params.neverTreatAsAfkFilter).replace(/\\\\/g, '\\'): undefined;
```

Comment:
Why `JSON.stringify` here? Seems unnecessary as it's already a string?

Reply:
my bad, yes its already a string. Was lazy and duplicating the previous line.